Update perf-data.adoc (#4848)
OpenNMS/opennms: docs/modules/operation/pages/events/perf-data.adoc

 AREA:octIn#73d216: \
 LINE1:octIn#4e9a06:"In " \
 GPRINT:octIn:AVERAGE:"Avg  \\: %8.2lf %s"
----

You can view the resulting performance graphs in menu:Reports[Resource Graphs].

[[resource-types]]
=== Create resource-types (optional)

If your time series data type is not `nodeSnmp` or `interfaceSnmp`, you need to create a resource-type file in `$\{OPENNMS_HOME}/etc/resource-types.d`.
Make sure you match the name with the event definition's `resourceType`.

> There is also `$\{instance}` variable available that can be used to enrich the resource type's label.

Reviewer: ```suggestion
There is also `$\{instance}` variable available that you can use to enrich the resource type's label.
```
